### PR TITLE
Refactor: Update build.gradle and fix import order in SelJodaDateTimePropertyTest to pass Spotless check

### PR DESCRIPTION
**Pull request by HankGuo93**
Pull Request type
----
- [ ] Bugfix
- [ ] Feature
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes (Please run `./gradlew build --write-locks` to refresh dependencies)
- [ ] Other (please describe):

**NOTE**: Please remember to run `./gradlew spotlessApply` to fix any format violations.

Changes in this PR
----
chore: Update build.gradle
- If the developer's computer environment is not set to English, running Unit Tests may result in errors.

fix: Correct import order to pass Spotless check in SelJodaDateTimePropertyTest
- Adjusted the order of imports to comply with Spotless formatting rules.
- Ran `./gradlew :netflix-sel:spotlessApply` to automatically fix format violations.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -39,7 +39,7 @@ allprojects {
     }
 
     test {
-        jvmArgs += ["-Djava.security.manager=allow"]
+        jvmArgs += ["-Djava.security.manager=allow", "-Duser.language=en", "-Duser.country=US"]
```
Comment (jun-he) [inline build.gradle]: Wondering if country is required?

**Comment by HankGuo93** [comment]
If your application works correctly without -Duser.country=US, then specifying the country is not mandatory. However, if your application relies on specific locale settings (such as date and currency formats), it is better to explicitly specify the country.

To ensure consistent behavior, it is advisable to set both the language and the country.

**Comment by jun-he** [comment]
@HankGuo93 can you remove the change in test jvmArgs from this PR and I can merge the other change about the `import`? Thanks.

**Comment by HankGuo93** [comment]
I've removed the changes to the test JVM arguments. The PR now only includes the import changes. Thanks!

**Review by jun-he — APPROVED** [review]
LGTM. thanks.

**Pull request merged**